ee/server/calendar/google_calendar.go

```diff
@@ -30,7 +30,7 @@ import (
 //    to create multiple events in the same calendar. This is useful for load testing. For example: john+test@example.com becomes john@example.com
 
 const (
-	eventTitle  = "💻🚫Downtime"
+	eventTitle  = "💻 🚫 Scheduled maintenance"
```

Comment:
I think we want to remove the space between the emojis:
```suggestion
	eventTitle  = "💻🚫 Scheduled maintenance"
```